Production_scripts/Process_extracts/extract_care_home.R

```diff
@@ -0,0 +1,123 @@
+#####################################################
+# Care Home Extract
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input - Social Care Care Home Episodes
+# Description - Match on Client data and Costs data
+#####################################################
+
+
+# Load packages
+library(dplyr)
+library(createslf)
+library(phsmethods)
+library(lubridate)
+
+
+fyyear <- check_year_format("1920")
+
+
+# Read in data---------------------------------------
+
+source_ch_data <- haven::read_sav(get_sc_ch_episodes_path(ext = "zsav")) %>%
+  # select episodes for FY
+  filter(record_keydate1 %in% range(start_fy(fyyear), end_fy(fyyear)) |
+    (record_keydate1 <= end_fy(fyyear) & record_keydate2 >= start_fy(fyyear) | is.na(record_keydate2))) %>%
+  # remove any episodes where the latest submission was before the current year
+  filter(convert_fyyear_to_year(fyyear) > substr(sc_latest_submission, 1, 4))
+
+
+# Match on Client Data ---------------------------------------
+
+# read client data in
+client_data <- readr::read_rds(get_source_extract_path(year, type = "Client", ext = "rds"))
```

Comment:
rds should be the default?
```suggestion
client_data <- readr::read_rds(get_source_extract_path(year, type = "Client"))
```